src/itemlistformaction.cpp

```diff
@@ -408,7 +408,7 @@ bool ItemListFormAction::process_operation(Operation op,
 		if (!show_searchresult) {
 			LOG(Level::INFO,
 				"ItemListFormAction: reloading current feed");
-			v->get_ctrl()->get_reloader()->reload(pos);
+			v->get_ctrl()->get_reloader()->start_reload_thread(pos);
 			invalidate_everything();
```

Comment:
We will probably have to call `invalidate_everything()` and force an STFL redraw when the reload finishes.
I'm not yet sure how to properly do that:
- not sure if STFL can be safely instructed to redraw from a background thread
- the ItemListFormAction might have been closed during the reload

Reply:
FWIW, `Reloader` already updates current formaction's `msg`, ignoring the first point and circumventing the second one.